etc/kayobe/stackhpc-overcloud-host-images.yml

```diff
@@ -0,0 +1,28 @@
+---
+##############################
+# Overcloud host image sources
+
+# WIP - currently unused
+# CentOS stream 8 overcloud host image source
+stackhpc_centos_8-stream_overcloud_host_image_url: "{{ stackhpc_release_pulp_content_url }}/kayobe-images/{{ openstack_release }}/centos/8-stream/{{ stackhpc_centos_overcloud_host_image_version }}/overcloud-centos-8-stream.qcow2"
```

Comment:
Version variable used here doesn't match the variable name below. Same with others below.

Reply:
Removed